loopchain/peer/block_manager.py

```diff
@@ -765,44 +764,55 @@ def vote_unconfirmed_block(self, block_hash, is_validated):
         )
         self.__channel_service.broadcast_scheduler.schedule_broadcast("VoteUnconfirmedBlock", block_vote)
 
-    def vote_as_peer(self):
-        """Vote to AnnounceUnconfirmedBlock
-        """
-        if self.__unconfirmedBlockQueue.empty():
-            return
-
-        unconfirmed_block: Block = self.__unconfirmedBlockQueue.get()
-        logging.debug(f"we got unconfirmed block ....{unconfirmed_block.header.hash.hex()}")
-
+    async def _vote(self, unconfirmed_block: Block):
         my_height = self.__blockchain.block_height
-        if my_height < (unconfirmed_block.header.height - 1):
+        if my_height < (unconfirmed_block.header.height - 2):
             self.__channel_service.state_machine.block_sync()
+            util.logger.debug(f"trigger block sync in _vote my_height({my_height}), "
+                              f"unconfirmed_block.header.height({unconfirmed_block.header.height})")
             return
 
         # a block is already added that same height unconfirmed_block height
         if my_height >= unconfirmed_block.header.height:
+            util.logger.debug(f"block is already added my_height({my_height}), "
+                              f"unconfirmed_block.header.height({unconfirmed_block.header.height})")
             return
 
-        logging.info("PeerService received unconfirmed block: " + unconfirmed_block.header.hash.hex())
-
-        block_version = self.__blockchain.block_versioner.get_version(unconfirmed_block.header.height)
-        block_verifier = BlockVerifier.new(block_version, self.__blockchain.tx_versioner)
-        block_verifier.invoke_func = self.__channel_service.score_invoke
-
-        reps = self.__channel_service.get_rep_ids()
-        exception = None
+        _exception = None
```

Comment:
How about using `exc = None`
Note that PEP8 recommends using trailing under score on conflicting with reserved keyword.
Of course it is not the case of reserved keyword.